docs/running-gil-disabled.md

```diff
@@ -32,6 +32,12 @@ Python 3.13.0b1+ experimental free-threading build (heads/3.13:d93c4f9, May 21 2
 
 Verify that the GIL is disabled at runtime with the following incantation:
 
+```bash
+python -c "import sysconfig; print(bool(sysconfig.get_config_var('Py_GIL_DISABLED')))"
+```
+
+or alternatively with:
+
```

Comment:
How about this?

```suggestion
This will be `True` on the free-threaded build when the GIL is re-enabled at runtime, but should be `False` before importing any packages. Note that `sys._is_gil_enabled()` is only available on Python 3.13 and newer, you will see an `AttributeError` on older Python versions.

You can check for a free-threaded build inside of Python with:

```bash
python -c "import sysconfig; print(bool(sysconfig.get_config_var('Py_GIL_DISABLED')))"
```

This will always be `True` on the free-threaded build and will be `False` otherwise, even on older Python versions. This should agree with the output of `python -VV`.
```

Reply:
That sounds about right. I'll check this locally and will apply it, with a minor tweak if needed.

Reply:
Made an update now, largely along these lines.